src/agenda.ts

```diff
@@ -8,25 +8,29 @@ export class Contact {
         this.surname=theSurname;
         this.address=theAddress;
     }
-    
-    // public displayContact(surname){
-    //     this.surname=surname;
-    //     return this.name + ' ' + this.surname + ' ' + this.address;
-    // }
 
 }
 
 
 export class Agenda {
     contacts: Contact[] = [];
```

Comment:
In this case we don't have derived classes and it needs to be accessed from other places so I would say public.